crates/turbopack-ecmascript/src/references/esm/export.rs

```diff
@@ -237,15 +237,18 @@ async fn handle_star_reexports(
 }
 
 #[turbo_tasks::value]
-struct ExpandResults {
-    star_exports: Vec<String>,
-    has_dynamic_exports: bool,
+pub struct ExpandResult {
+    pub esm_ref: Vc<Box<dyn ModuleReference>>,
+    pub root_module: Vc<Box<dyn EcmascriptChunkPlaceable>>,
```

Comment:
It's weird that `expand_star_exports` returns its input arguments again

Reply:
yeah I added it back because the `CodeGenerateable` depends pretty heavily on it, I'll see if I can refactor it somehow